src/licensedcode/data/rules/ibm_developerworks.yml

```diff
@@ -0,0 +1,2 @@
+licenses:
+    - ibm-developerworks-community-download

```

Comment:
How did you conclude this was this license?
It looks to me as a plain proprietary notice